### PR TITLE
Update Style for Generating Reference Model Anchors at Build Time

### PR DESCRIPTION
**Pull request by JustKuzya**
# Committer Notes
Partial fix for usnistgov/issue#1681

These two files hoos up anchors-js stuff for the older style deep-links at the recommended time in the event-lifecycle
Also makes sure that CSS hooks up behaviors to old site and ignores the newer versions.

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~

### CODE REVIEW

File: docs/assets/scss/schema-docs.scss [hunk 1]
```diff
@@ -330,4 +330,34 @@
...
+  font-variation-settings: normal;
+  font-weight: normal;
+  font-stretch: normal;
+  font-size: 1em;
+  line-height: inherit;
+  font-family: anchorjs-icons;
+  position: absolute;
+  margin-left: -1em;
+  padding-right: 0.5em;
+}
```
Comment (aj-stein-nist) [inline docs/assets/scss/schema-docs.scss:352]: Blocking: can we make the anchor class a name more intuitive than `no-anchor-xslt` as that explains implementations but not the purpose (so only useful and relevant for a little bit, especially given the no-anchor). Also we agreed the comments and `font-family` declaration isn't effective or necessary, so can we just remove it?

Are the other class attributes still necessary for styling or just mimicking the AnchorJS implementation we will be removing as a consequence of this change?

```suggestion
.reference-element-anchor {
  font-style: normal;
  font-variant: normal;
  font-kerning: auto;
  font-optical-sizing: auto;
  font-feature-settings: normal;
  font-variation-settings: normal;
  font-weight: normal;
  font-stretch: normal;
  font-size: 1em;
  line-height: inherit;
  position: absolute;
  margin-left: -1em;
  padding-right: 0.5em;
}
```

File: docs/assets/scss/schema-docs.scss [hunk 2]
```diff
@@ -330,4 +330,34 @@
...
+
+.no-anchor-xslt:hover {
+  text-decoration: underline;
+  transform: scale(1.06);
+  border-image: "⚓";
+}
```
Comment (aj-stein-nist) [inline docs/assets/scss/schema-docs.scss:358]: Blocking: expanding the above recommendation.

```suggestion
.reference-element-anchor:hover {
  text-decoration: underline;
  transform: scale(1.06);
}
```

File: docs/assets/scss/schema-docs.scss [hunk 3]
```diff
@@ -330,4 +330,34 @@
...
+
+.no-anchor-xslt :hover::before {
+  font-family: anchorjs-icons;
+  background-image: "⚓";
 }
```
Comment (aj-stein-nist) [inline docs/assets/scss/schema-docs.scss:363]: Blocking: expanding the above recommendation.

```suggestion

```

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
Excellent work, just some important but very minor feedback to make the class names more legible to me. Also, I have some follow-up questions about layover AnchorJS styles we will be removing.

File: docs/assets/scss/schema-docs.scss (hunk 3 above)
Comment (JustKuzya) [reply docs/assets/scss/schema-docs.scss:363]: I knew that font-family: anchorjs-icons; would include ⚓. Whatever family we use by default and includes ⚓ is fine with me.

Comment (aj-stein-nist) [reply docs/assets/scss/schema-docs.scss:363]: > I knew that font-family: anchorjs-icons; would include ⚓. Whatever family we use by default and includes ⚓ is fine with me.

I think we have discussed this before: why do we need the anchor? It does not render with the accompanying Metaschema PR and we will re-generate all documentation for other released OSCAL reference docs, so we do not need this anymore (even as an interim gap measure). Am I remembering correctly?

**Comment by JustKuzya** [comment]
I thought that I pushed it either Monday last week or Friday the week before. I probably need to map the changes to the Issue from github
DC

**Review by aj-stein-nist — DISMISSED** [review]

**Review by JustKuzya — COMMENTED** [review]
The "hooked up" apparently is just one right-hand ring-finger flutter away from the "hooled up" :) My bad for not catching it before push

**Review by aj-stein-nist — APPROVED** [review]

**Comment by aj-stein-nist** [comment]
Sigh, how can we even get a merge conflict on pushing up a submodule!? I will have to investigate later.

**Comment by aj-stein-nist** [comment]
The local performance metrics were dubious when run in a virtualized developer weird ways but Dmitry provided interim metrics ([Site-Benchmarks.xlsx](https://github.com/usnistgov/OSCAL/files/11748576/Site-Benchmarks.xlsx)). Running `hugo serve` locally seems to have weird crashes even with Chrome Lighthouse running locally in weird ways I also encountered, so more to follow on that as we close out this issue.

**Pull request merged**